.pre-commit-config.yaml

```diff
@@ -1,28 +1,28 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.2.3
+    rev: v3.3.0
     hooks:
+      - id: trailing-whitespace
       - id: no-commit-to-branch
       - id: check-docstring-first
       - id: end-of-file-fixer
       - id: check-merge-conflict
+      - id: check-yaml
 
-  # isort should run before black as black sometimes tweaks the isort output
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.5.3
+  - repo: https://github.com/ambv/black
```

Comment:
Should this be: ?
```
  - repo: https://github.com/psf/black
    rev: stable
```

Reply:
cc. @andersy005

Reply:
Both should work because if you go to https://github.com/ambv/black, GitHub will redirect you to https://github.com/psf/black

Reply:
Cool. Understand performance wise doesn't matter but would prefer the latest link. Same way I would prefer people to reference xarray-contrib/xskillscore over raybellwaves/xskillscore.

Reply:
Just updated to @raybellwaves suggestion.